docs/installation/Binaries.md

```diff
@@ -63,7 +63,7 @@ please consult their documentation.
 
 You can set up Nussknacker as a systemd service using our example unit file.
 
-1. Download distribution as described in [Binary package installation](#binary-package-installation)
+1. Download distribution as described in [Binary package installation](.)
```

Comment:
Can we do it the same way as https://github.com/TouK/nussknacker/pull/6706/files#diff-ed2bd6b2676087ad5062b8cb0095b79810a54959cf53ab32b8f182bb24090286R66 ?